new/AksumiteStyle.xml

```diff
@@ -40,7 +40,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <profileDesc>
             <creation/>
             <abstract>
-                <p/>
+                <p>Style of paintings used approximately during the Christian Aksumite period.</p>
```

Comment:
All is fine, but I would suggest, now that you are getting back into the flow, to always add (in this and the other records) a `<change>` ellement with the date, your name and a short description of the changes you carried out (e.g. "added abstract/begun description of painting style" - you know best), so that everything is documented and later citation and attribution issues won't come up